spellings/action.yml

```diff
@@ -19,105 +19,105 @@ inputs:
-        sudo apt-get install npm -y
-        sudo npm install -g cspell
-
-        # Add the Github Action Path to PATH
-        export PATH="$GITHUB_ACTION_PATH:$PATH"
-
-        # Account for starting with an alternate path in a repository
-        # Do this by copying the cspell config and wordlist to the desired path
-        # Wrap in a set +e so Github doesn't throw an error if the file or
-        # directory already exists.
-        set +e
-        cp cspell.config.yaml ${{ inputs.path }}
-        mkdir ${{ inputs.path }}/.github
-        cp .github/.cSpellWords.txt ${{ inputs.path }}/.github
-        cd ${{ inputs.path }}
-        set -e
-
-        echo -e "::endgroup::"
-        # Make sure we have all the commands we need.
-        echo -e "${{ env.bashInfo }} fdfind --version $(fdfind --version) ${{ env.bashEnd }}"
-        echo -e "${{ env.bashInfo }} cspell --version $(cspell --version) ${{ env.bashEnd }}"
-
-        # Only reach this line if no errors were hit above
+  - env:
+      bashPass: \033[32;1mPASSED -
+      bashInfo: \033[33;1mINFO -
+      bashFail: \033[31;1mFAILED -
+      bashEnd:  \033[0m
+      stepName: Set-Up The Spell Checker
+    name: ${{ env.stepName }}
+    id: spell-checker-setup
+    shell: bash
+    run: |
+      # ${{ env.stepName }}
+      echo -e "::group::${{ env.bashInfo }} ${{ env.stepName }} ${{ env.bashEnd }}"
+
+      # Install the Dependencies we need to run the spell-checker
+      sudo apt-get install util-linux -y
+      sudo apt-get install fd-find -y
+      sudo apt-get install npm -y
+      sudo npm install -g cspell
+
+      # Add the Github Action Path to PATH
+      export PATH="$GITHUB_ACTION_PATH:$PATH"
+
+      # Account for starting with an alternate path in a repository
+      # Do this by copying the cspell config and wordlist to the desired path
+      # Wrap in a set +e so Github doesn't throw an error if the file or
+      # directory already exists.
+      set +e
+      cp cspell.config.yaml ${{ inputs.path }}
+      mkdir ${{ inputs.path }}/.github
+      cp .github/.cSpellWords.txt ${{ inputs.path }}/.github
+      cd ${{ inputs.path }}
+      set -e
+
+      echo -e "::endgroup::"
+      # Make sure we have all the commands we need.
+      echo -e "${{ env.bashInfo }} fdfind --version $(fdfind --version) ${{ env.bashEnd }}"
+      echo -e "${{ env.bashInfo }} cspell --version $(cspell --version) ${{ env.bashEnd }}"
+
+      # Only reach this line if no errors were hit above
+      echo -e "${{ env.bashPass }} ${{ env.stepName }} ${{ env.bashEnd }}"
+
+  - env:
+      bashPass: \033[32;1mPASSED -
+      bashInfo: \033[33;1mINFO -
+      bashFail: \033[31;1mFAILED -
+      bashEnd:  \033[0m
+      stepName: Spell Checker
+    name: ${{ env.stepName }}
+    id: run-spell-checker
+    working-directory: ${{ inputs.path }}
+    shell: bash
+    run: |
+      # ${{ env.stepName }}
+      echo -e "::group::${{ env.bashInfo }} ${{ env.stepName }} ${{ env.bashEnd }}"
```

Comment:
Fix bug where the echo group doesn't work since it's missing the semicolon
Example of bug [here](https://github.com/FreeRTOS/FreeRTOS/actions/runs/6477161501/job/17586919705#step:3:1241)